contracts/VoucherHub.sol

```diff
@@ -66,14 +66,15 @@ contract VoucherHub is
     }
 
     function initialize(
```

Comment:
Eventually notify clients before merging

Reply:
Yes good point, hopefully that won't break their work, it's just the ABI that change but they never call this function